crates/auto-hash-map/src/set.rs

```diff
@@ -22,6 +22,20 @@ impl<K, H> Default for AutoSet<K, H> {
     }
 }
 
+impl<K: Display, H> Display for AutoSet<K, H> {
```

Comment:
Is this impl still needed? It looks like you're iterating over the set manually now in `execution_result`.

Reply:
You're right, done.